{{cookiecutter.project_slug}}/ios/fastlane/Fastfile

```diff
@@ -9,21 +9,17 @@
 #
 #     https://docs.fastlane.tools/plugins/available-plugins
 #
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
-
 opt_out_usage
 
 default_platform(:ios)
 
-app_name = "{{cookiecutter.project_slug}}"
+app_name = "{{cookiecutter.project_dash_slug}}"
 base_name = "{{cookiecutter.project_slug}}"
 team_name = "Crowdbotics Corporation"
 team_id = "6YKR59QXKM"
 filename = "#{base_name}.xcodeproj"
 workspaceName = "#{base_name}.xcworkspace"
-identifier = "com.crowdbotics.#{app_name}"
+identifier = "com.crowdbotics.#{base_name}"
```

Comment:
The identifier was changed everywhere to use underscores, so it needed to be changed to base_name here.